model_servers/llamacpp_python/Makefile

```diff
@@ -12,10 +26,23 @@ mistral-7b-instruct-v0.1.Q4_K_M.gguf:
 install:
 	pip install -r tests/requirements.txt
 
-.PHONY: run
-run:
-	podman run -it -d -p 8001:8001 -v ./models:/locallm/models:ro,Z -e MODEL_PATH=models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=8001 --net=host ghcr.io/redhat-et/model_servers
+.PHONY: run-linux
+run-linux:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models:ro,Z -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) --net=host $(IMAGE); cd model_servers/llamacpp_python;
+
+.PHONY: run-darwin
+run-darwin:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) --net=host $(IMAGE); cd model_servers/llamacpp_python;
+
+.PHONY: run-linux-local
+run-linux-local:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models:ro,Z -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT) $(IMAGE); cd model_servers/llamacpp_python;	
+
+.PHONY:  run-darwin-local
+run-darwin-local:
+	cd ../..; podman run -it -d -p $(PORT):$(PORT) -v ./models:/locallm/models -e MODEL_PATH=/locallm/models/mistral-7b-instruct-v0.1.Q4_K_M.gguf -e HOST=0.0.0.0 -e PORT=$(PORT)  $(IMAGE); cd model_servers/llamacpp_python;
+
 
 .PHONY: test
 test:
-	pytest --log-cli-level NOTSET
+	pytest --log-cli-level NOTSET
```

Comment:
Should the tests also do a make all on this Makefile?

Reply:
we can add in a follow-up @lmilbaum

Reply:
I have added a version of this, let me know if I should remove it